editor/app/fatalities/[record_locator]/page.tsx

```diff
@@ -57,7 +68,57 @@ export default function FatalCrashDetailsPage({
             {data[0].law_enforcement_ytd_fatality_num}
           </span>
         </div>
-      )}
+      </Row>
+      <Row>
+        <Col>
+          <Card>
+            <Card.Body>
+              <Table>
+                <tbody>
+                  <tr>
+                    <td style={{ textWrap: "nowrap" }} className="fw-bold">
+                      Date
+                    </td>
+                    <td>{formatIsoDateTimeWithDay(crash.crash_timestamp)}</td>
+                  </tr>
+                  <tr>
+                    <td style={{ textWrap: "nowrap" }} className="fw-bold">
+                      Units involved
+                    </td>
+                    <td></td>
+                  </tr>
+                  <tr>
+                    <td style={{ textWrap: "nowrap" }} className="fw-bold">
+                      Collision type
+                    </td>
+                    <td>{crash.collsn?.label}</td>
+                  </tr>
+                  <tr>
+                    <td style={{ textWrap: "nowrap" }} className="fw-bold">
+                      Roadway owner
+                    </td>
+                    <td>
+                      {crash.is_coa_roadway === true
+                        ? "City of Austin"
+                        : "TxDOT"}
+                    </td>
+                  </tr>
+                </tbody>
```

Comment:
This looks good to me. If we need to repeat this pattern again, I think we should extract a `DataCardBody` component from the `DataCard` component and drop that in as a replacement.